.jenkins/pytorch/test.sh

```diff
@@ -386,10 +391,11 @@ elif [[ "${BUILD_ENVIRONMENT}" == *-test1 || "${JOB_BASE_NAME}" == *-test1 ]]; t
   fi
   install_torchvision
   test_python_shard1
+  test_aten
+  test_numpy_uninstall
```

Comment:
good to add a comment saying this should always be the last test suite to execute - to avoid future changes appending new tests behind.

or add `conda install numpy` at the end of the `test_numpy_uninstall()` function is also a good solution?

Reply:
I've modified the script as @rgommers suggested to use fake_numpy rather than uninstall an actual one